frontend/packages/dev-console/integration-tests/features/addFlow/create-from-docker-file.feature

```diff
@@ -10,17 +10,20 @@ Feature: Create Application from Docker file
 
         @regression
         Scenario: Dockerfile details after entering git repo url: A-05-TC01
-            Given user is on Import from Docker file page
+            Given user is on Import from Git page
              When user enters docker git url as "https://github.com/sclorg/nodejs-ex.git"
```

Comment:
```suggestion
            Given user is on Import from Git form
             When user enters docker git url "https://github.com/sclorg/nodejs-ex.git"
```

Same for all the statements below

Reply:
This file got missed while pushing changes, updated